CMakeLists.txt

```diff
@@ -37,6 +36,15 @@ endif()
 list (APPEND CMAKE_MODULE_PATH
   "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
+# Getting git SHA-1
+list(APPEND CMAKE_MODULE_PATH 
+  "${CMAKE_CURRENT_SOURCE_DIR}/getGitRevisionModule/")
```

Comment:
move all cmake related files into sub-directory of `cmake` to avoid polluting top-level directory.

Reply:
solve in https://github.com/seahorn/seahorn/pull/183/commits/c10ab9118b728936ed952df355e328e92521a22a